dev/ProgressBar/TestUI/ProgressBarPage.xaml.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Markup;
+using Windows.UI;
+using System.Windows.Input;
+
+using ProgressBar = Microsoft.UI.Xaml.Controls.ProgressBar;
+//using Windows.UI.Xaml.Tests.MUXControls.ApiTests.RepeaterTests.Common;
+using Windows.UI.Xaml.Data;
+using System.Numerics;
+using Windows.UI.Xaml.Shapes;
+
+namespace MUXControlsTestApp
+{
+    [TopLevelTestPage(Name = "ProgressBar")]
+    public sealed partial class ProgressBarPage : TestPage
+    {
+        public ProgressBarPage()
+        {
+            this.InitializeComponent();
+            Loaded += ProgressBarPage_Loaded;  
+        }
+
+        private void ProgressBarPage_Loaded(object sender, RoutedEventArgs e)
+        {
+            TestProgressBar.Loaded += TestProgressBar_Loaded;
+            Loaded -= ProgressBarPage_Loaded;
+        }
+
+        private void TestProgressBar_Loaded(object sender, RoutedEventArgs e)
```

Comment:
>TestProgressBar_Loaded [](start = 21, length = 22)

Page_loaded should be all you need here, I think we added this when we weren't sure what was going on with the GetVisualStateGroups failing.